src/app/helpers/announcement/controller.js

```diff
@@ -86,20 +86,25 @@ class AnnouncementController {
 
       try {
         var newAnnouncementObj = await (this.__createAnnouncement(requestObj.body.request))
-        //TODO: sent count as async process
       } catch (error) {
         throw { msg: 'unable to process the request!', statusCode: HttpStatus.INTERNAL_SERVER_ERROR }
       }
 
-      try {
-        //TODO: notification: incomplete implementation 
-        await (this.__createAnnouncementNotification( /*announcement data*/ ))
-        return { announcement: newAnnouncementObj.data }
-      } catch (e) {
+    try {
```

Comment:
This try catch should not be here. response to create request should be independent of the outcome of notification call.